pkg/cmd/cli/cmd/rsync.go

```diff
@@ -0,0 +1,381 @@
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/golang/glog"
+	"github.com/spf13/cobra"
+	"gopkg.in/pipe.v2"
```

Comment:
The main reason was convenience. There was less setup required with this library. Not a big deal to go back to os.Pipe.